### ISSUE TITLE
config.transport shouldn't be required

### ISSUE THREAD

**Issue by hugodes** [niftylettuce/email-templates#264]
We've been using email-templates in our project for some time, and I noticed that since v3, the transport config object is required.
We only use email-templates to generate the body of the email and we use our own transport method with the result of the render method.
For these types of use-cases could we make the transport object in the conf optional ?

**Comment by niftylettuce** [comment]
I agree, will take this out soon

**Comment by yelled3** [comment]
@hugodes agreed - I'm sticking to v2 for now :-)

**Comment by niftylettuce** [comment]
Just patched this and released version 3.2.1 to npm.

```sh
npm install email-templates@latest
```

or if you're using yarn

```
yarn add email-templates @latest
```

**Comment by yelled3** [comment]
@niftylettuce thanks for the quick response!
Also, it would be great if you can update the docs with an example for how to use `render` without sending.

Thanks again!

**Comment by niftylettuce** [comment]
@yelled3 I've exposed a new method called `renderAll` which should make it so much simpler if you're trying to render all the templates.  I've also documented a few examples for you too.  See <https://github.com/niftylettuce/email-templates#render-html-andor-text> for these examples.

Let me know if you have any questions.  This is available now in npm as v3.3.2:

```sh
npm install email-templates@latest
```

or with yarn:

```sh
yarn add email-templates@latest
```
